crates/cli/src/subcommands/dns.rs

```diff
@@ -30,49 +31,34 @@ fn get_subcommands() -> Vec<Command> {
                     .required(true)
                     .help("The top level domain that you would like to register"),
             )
-            .arg(Arg::new("identity").long("identity").short('i').help(
-                "The identity that should own this tld. If no identity is specified, then the default identity is used",
-            ))
             .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The nickname, host name or URL of the server on which to register the domain"),
+                common_args::identity()
+                .help("The identity that should own this tld. If no identity is specified, then the default identity is used"),
             )
+            .arg(common_args::server())
             .after_help("Run `spacetime dns register-tld --help` for more detailed information.\n"),
         Command::new("lookup")
             .about("Resolves a domain to a database address")
             .arg(Arg::new("domain").required(true).help("The name of the domain to lookup"))
-            .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The nickname, host name or URL of the server on which to look up the domain name"),
-            )
+            .arg(common_args::server())
             .after_help("Run `spacetime dns lookup --help` for more detailed information"),
         Command::new("reverse-lookup")
             .about("Returns the domains for the provided database address")
             .arg(Arg::new("address").required(true).help("The address you would like to find all of the known domains for"))
-            .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The nickname, host name or URL of the server on which to look up the address"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.